tests/wms_test_utils.py

```diff
@@ -61,3 +61,51 @@ class WmsServiceDefault(BaseWmsService):
     def ping(self, pass_thru):
         _LOG.info(f"DEFAULT {pass_thru}")
         return 0, "default"
+
+
+class WmsServiceFromCmdline(BaseWmsService):
+    """WMS service class with its own default settings."""
+
+    @property
+    def defaults(self):
+        return {"corge": 1}
```

Comment:
I probably would have made the values a unique string (e.g., "cmdline") so that it wouldn't be as easy to make a mistake in the tests.